apps/telehealth-widget/HOW-THE-DEMO-WORKS.md

```diff
@@ -0,0 +1,37 @@
+
+
+## Check in
+
+The solution lists Amazon Chime SDK messaging channels moderated by the user. The solution sends messages to the other user using the [AWS SDK v3 sendChannelMessageCommand](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-chime-sdk-messaging/classes/sendchannelmessagecommand.html) to track the presence and update the user status as “Checked In” or “Added”. The solution listens to these messages using the MessagingSession and accordingly updates the UI with the user status and enable/disable audio-video calling feature.
+
+
+## SMS notification
+
+If a doctor or a patient is not present (in “Added” status) at the appointment time, they receive a text message alerting to check-in into their appointment. The appointment creation AWS Lambda function starts the [AWS step function](https://aws.amazon.com/step-functions/getting-started/) execution. The AWS step function waits for the appointment time and then it sends an SMS to the doctor and the patient in the appointment. The solution uses [Amazon Pinpoint](https://aws.amazon.com/pinpoint/) to send an SMS notification. We save the appointment date/time and doctor/patient information in the channel metadata.
+
+## Chat
+
+Upon check-in into the appointment, the doctor and patient see a chat widget where they can chat. The solution sets up separate messaging channels for each doctor-patient appointment. In this chat widget, the doctor can see an option to place a video and a PSTN outbound call invite to the patient.
+
+## Video calling
+
+The solution uses [Amazon Chime SDK React Component Library](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/introduction--page), [Amazon Chime SDK for JavaScript](https://github.com/aws/amazon-chime-sdk-js) for audio-video WebRTC based calling. The solution calls lambda functions which call `CreateMeeting`, `CreateAttendee` Amazon Chime SDK APIs to receive the meeting and attendee information and then sets up the `MeetingProvider` to join a meeting. The doctor side sends a meeting invite over the messaging channel and awaits the response to join a meeting. If the patient accepts in the meeting invite, the solution creates the attendee for the same meeting and then both doctor and the patient join the same meeting in a separate meeting widget.
```

Comment:
- `WebRTC based calling` => `WebRTC-based calling`
- `the patient accepts in the meeting invite` => `the patient accepts the meeting invite` (Remove `in`)
- `the same meeting and then both doctor` => `the same meeting, and then both doctor` (Add `,`)

Reply:
Thanks for the detailed feedback! Fixed as suggested.